examples/scriptcheck.sh

```diff
@@ -12,19 +12,22 @@ Usage: $SCRIPT [-h] [-d]
           [-e] Display excerpt of error message on failure
           [-d] Skip tests involving additional data downloads
           [-t] Skip tests related to learned model training
+          [-g] Skip tests that need the GPU
```

Comment:
Major nitpick: "a GPU" would be better. Ignore this comment if no other changes need to be made.